src/scss/components/_tooltip.scss

```diff
@@ -19,3 +20,55 @@
     display: flex;
   }
 }
+
+.utrecht-form-field__label .tooltip {
+  --#{prefix(tooltip-arrow-width)}: 1rem;
+  --#{prefix(tooltip-arrow-height)}: 0.5rem;
+  --#{prefix(tooltip-bg)}: var(--{prefix(color-bg)});
```

Comment:
I'm probably holding this wrong, my design-token-fu is non existent. :pray: :bow: